data/json/monsters.json

```diff
@@ -2003,6 +2003,36 @@
     "flags":["SEES", "HEARS", "SMELLS", "STUMBLES", "WARM", "BASHES", "GROUP_BASH", "POISON", "HARDTOSHOOT", "NO_BREATHE", "REVIVES", "BONES", "PUSH_MON"]
     },{
     "type" : "MONSTER",
+    "id" : "mon_zombie_shady",
+    "name": "shady zombie",
+    "species":"ZOMBIE",
+    "default_faction":"zombie",
+    "symbol":"Z",
+    "color":"ltgray",
```

Comment:
Is ltgray the same color as shadows in ASCII? :) I just copied it from the smoker.. If nobody else has a suggestion I'll probably just pick a random color.